core/core-spacemacs.el

```diff
@@ -73,7 +73,8 @@
   (configuration-layer/initialize)
   ;; default theme
   (let ((default-theme (car dotspacemacs-themes)))
-    (spacemacs/load-theme default-theme)
+    (unless (string= "default" (car dotspacemacs-themes))
```

Comment:
I think `(eq 'default default-theme)` is better. It's already let-bound so you don't have to take the `car` again, and it's a symbol, so best to use the correct equality function.